super_dpf.py

```diff
@@ -0,0 +1,129 @@
+import json
+import os
+import sys
+import collections
+
+
+class SuperDPF(object):
+    PATHS = {'project': '/home/pi/SuperDPF'}
+    PATHS['conf_template'] = '{}/.conf_template.json'.format(PATHS['project'])
+    PATHS['config'] = '{}/config.json'.format(PATHS['project'])
+    PATHS['photos'] = '{}/sdpf_photos'.format(PATHS['project'])
+
+    STORAGE_TYPES = ['aws', 'gphotos']
+
+    def __init__(self):
+        if self.is_first_run:
+            try:
+                open(self.PATHS['config'], 'r')
+            except:
+                with open(self.PATHS['conf_template'], 'r') as template:
+                    template = json.load(template)
+                    template['first_run'] = ''
+                    with open(self.PATHS['config'], 'w') as config_file:
+                        json.dump(template, config_file)
+
+            self.create_photo_dirs()
+            self.set_storage_types()
+        else:
+            self.sync()
+
+    @property
+    def settings(self):
+        try:
+            with open(self.PATHS['config'], 'r') as config_file:
+                config = json.load(config_file)
+            return config
+        except IOError:
+            print 'There was an error loading the settings file'
+            return
```

Comment:
This exception can happen for more reasons than the file not existing.
os.path.isfile(filename) is useful for defensively checking that the file is there and is a file (not a directory or pipe etc...)
I would suggest extracting the error text from the exception and passing it back to the user.